llvm/lib/Target/PowerPC/PPCISelLowering.cpp

```diff
@@ -15686,6 +15690,50 @@ static SDValue getSToVPermuted(SDValue OrigSToV, SelectionDAG &DAG,
                      OrigSToV.getOperand(0));
 }
 
+static bool isShuffleMaskInRange(const SmallVectorImpl<int> &ShuffV,
+                                 int HalfVec, int LHSLastElementDefined,
+                                 int RHSLastElementDefined) {
+  for (int I : seq<int>(0, ShuffV.size())) {
+    int Index = ShuffV[I];
```

Comment:
`I` is only used to get the element out of `ShuffV`. That can be done easier:
```suggestion
  for (int Index : ShuffV) {
```

Reply:
Great suggestion, thanks!